src/main/scala/viper/gobra/ast/internal/utility/Nodes.scala

```diff
@@ -139,6 +138,7 @@ object Nodes {
         case OptionSome(exp) => Seq(exp)
         case OptionGet(exp) => Seq(exp)
         case Negation(operand) => Seq(operand)
+        case BitNeg(operand) => Seq(operand)
```

Comment:
`BitNeg` is missing in `GobraStrategy.scala`. What about all other newly introduced internal nodes?

Reply:
Good catch, I often forget about these because `Node` is not a sealed trait and thus, the compiler does not complain about missing cases.

Reply:
I do also always forget them as well. We should either check whether we need this serialization & deserialization at all and then either remove it or write unit tests that consistently trigger them